scripts/delete.py

```diff
@@ -1,6 +1,6 @@
 import logging
 
-from scripts.util import documents
+from scripts.util import documents_v2 as documents
```

Comment:
this is actually a naming collision between the documents_v2 script and documents_v2 generator, which I should definitely resolve anyway.